calibr8/core.py

```diff
@@ -677,7 +677,9 @@ def load(cls, filepath: os.PathLike):
 
         # assign additional attributes (check keys for backwards compatibility)
         obj.theta_bounds = tuple(map(tuple, data["theta_bounds"])) if "theta_bounds" in data else None
-        obj.theta_guess = tuple(data["theta_guess"]) if "theta_guess" in data else None
+        obj.theta_guess = (
+            tuple(data["theta_guess"]) if "theta_guess" in data and data.get("theta_guess") != None else None
```

Comment:
```suggestion
            tuple(data["theta_guess"]) if "theta_guess" in data and data.get("theta_guess") is not None else None
```

Reply:
Ah 💡! So in python there is just one None object. That's good to know. Thanks!